backend/app/services/llm/providers/eai.py

```
@@ -0,0 +1,130 @@
+    LLMResponse,
+    Usage,
+)
+from app.services.llm.providers.base import BaseProvider
+
+
+logger = logging.getLogger(__name__)
+
+
+
+
+class BaseProvider:
```

Ruff (0.15.4)

[error] Redefinition of unused `BaseProvider` from line 20: `BaseProvider` redefined here

(F811)